src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/onestatementperline/InputOneStatementPerLineMultilineTwo.java

```diff
@@ -124,8 +37,8 @@ private void foo4() {
         for(int n = 0,
             k = 1
             ; n<5
-            ; ) { int a = 5,
-        b = 2;}
+                ; ) { int a = 5,
+                b = 2;}
```

Comment:
Please rever this minor change, if not required.

Reply:
all is done